RecoLocalFastTime/FTLCommonAlgos/plugins/FTLSimpleUncalibRecHitAlgo.cc

```diff
@@ -0,0 +1,51 @@
+#include "RecoLocalFastTime/FTLCommonAlgos/interface/FTLUncalibratedRecHitAlgoBase.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+class FTLSimpleUncalibRecHitAlgo : public FTLUncalibratedRecHitAlgoBase {
+ public:
+  /// Constructor
+  FTLSimpleUncalibRecHitAlgo( const edm::ParameterSet& conf,
+                              edm::ConsumesCollector& sumes ) : 
+  FTLUncalibratedRecHitAlgoBase( conf, sumes ) { 
+    uint32_t nBits     = conf.getParameter<uint32_t>("adcNbits");
+    double saturation  = conf.getParameter<double>("adcSaturation");
+    adcLSB_            = saturation/std::pow(2.,nBits);    
```

Comment:
I'm still not seeing why you're using `pow(2., nBits);` instead of shift. `nBits` is 12 according to one of the configs.